lib/screens/home_page/home_page.dart

```diff
@@ -15,50 +17,67 @@ class HomePage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final todoBloc = ToDoListBloc();
```

Comment:
We should use a stateful widget and call these functions from the initState function instead of putting them in the build function